src/main.ts

```diff
@@ -517,7 +513,7 @@ export class Purchases {
 
     const certainHTMLTarget = resolvedHTMLTarget as unknown as HTMLElement;
 
-    const asModal = !Boolean(htmlTarget);
+    const asModal = !htmlTarget;
```

Comment:
this one was automatically picked up and fixed, redundant boolean conversion rule